controllers/space/space_controller.go

```diff
@@ -474,6 +478,10 @@ func (r *Reconciler) ensureSpaceDeletion(ctx context.Context, space *toolchainv1
 	logger := log.FromContext(ctx)
 
 	logger.Info("terminating Space")
+	if err := r.setStatusTerminating(ctx, space); err != nil {
+		logger.Error(err, "error updating status")
```

Comment:
what about wrapping the underlying error and returning the new `err`? Otherwise, we'll get 2 lines in logs, which is harder to track :/

Reply:
I'm sorry, but I don't follow. 
What do you propose to do differently compared to what we currently have in the controller? Do you want to drop the line 482 (and wrap the error with the message instead)? If yes, then this change doesn't affect only this line, but all other lines in the whole controller and most likely in all other controllers as well. I only kept the consistency with the current code.
TBH, I'm fine with removing the duplication in the logs, but this is much bigger effort.

Reply:
sure, let's address that in a separate PR later, then